scripts/convert_hf_checkpoint.py

```diff
@@ -50,12 +48,9 @@ def convert_hf_checkpoint(
     sd_meta = model.state_dict()
     sd = {}
 
-    # Load the json file containing weight mapping
-    pytorch_bin_map_json_path = checkpoint_dir / "pytorch_model.bin.index.json"
-    with open(pytorch_bin_map_json_path) as json_map:
-        bin_index = json.load(json_map)
-
-    bin_files = set(el for el in bin_index["weight_map"].values())
+    bin_files = list(checkpoint_dir.glob("*.bin"))
+    if not bin_files:
+        raise ValueError(f"Expected {str(checkpoint_dir)!r} to contain .bin files")
```

Comment:
Simplifying, we do this in lit-parrot

Reply:
Seems OK per se, but I wonder if there might be instances where users have other bin files for no good reason (e.g. an older version of their model had a split into smaller files).

Reply:
So far I haven't seen any hf-hub checkpoint that has that.

Reply:
That's a good point. I'll revert it